### PR TITLE
rev 1.0

### PR DESCRIPTION
**Pull request by qed11**
use list to calculate work done in each member
need some values of the forces in the members
could go in deeper

### CODE REVIEW

**Comment by qed11** [comment]
btw don't merge too soon
we still need to make changes before we merge
add Ryan as a contributor too?

**Review by Cravat5386 — APPROVED** [review]

**Comment by Cravat5386** [comment]
Already added him

**Pull request merged**
